packages/client/test/events.spec.ts

```diff
@@ -68,29 +67,6 @@ describe('Events', () => {
       API.setProvider(myProvider);
     });
 
-    it('It defines a mechanism for signalling `PROVIDER_CONFIGURATION_CHANGED`', (done) => {
-      API['_apiEvents'] = new OpenFeatureEventEmitter();
-      const myProvider: Provider = {
-        metadata: {
-          name: 'mock-events',
-        },
-        initialize: jest.fn(() => {
-          return Promise.resolve(undefined);
-        }),
-        events: new OpenFeatureEventEmitter(),
-      } as unknown as Provider;
-      API['_apiEvents'].addListener(ApiEvents.ProviderChanged, () => {
-        try {
-          expect(API.providerMetadata.name).toBe(myProvider.metadata.name);
-          done();
-        } catch (err) {
-          done(err);
-        }
-      });
-      API.setProvider(myProvider);
-
-    });
-
```

Comment:
PROVIDER_CONFIGURATION_CHANGED is not for detecting when the provider itself is changed, it's for signalling to the SDK that a flag configuration has changed. The idea is that, for example, a CRD is changed in K8S, and the provider can check for that and run `this.events.emit(Provider.Changed)` and then the consuming application can get updates in realtime. `ApiEvents` is an implementation detail that doesn't need testing.